File: src/main/java/me/diax/diax/commands/information/Credits.java
package me.diax.diax.commands.information;

import me.diax.comportment.jdacommand.Command;
import me.diax.comportment.jdacommand.CommandAttribute;
import me.diax.comportment.jdacommand.CommandDescription;
import me.diax.diax.util.Embed;
import net.dv8tion.jda.core.entities.Message;

@CommandDescription(
        name = "credits",
        triggers = "credits",
        description = "Display the credits for Diax.",
        attributes = @CommandAttribute(key = "private")
)
public class Credits implements Command {

    @Override
    public void execute(Message message, String s) {
        message.getTextChannel().sendMessage(
                Embed.transparent()
                        .addField("comportment#4475", "Owner // Head-Developer", false)
                        .addField("NachtRaben#8307", "Developer // Music", false)
                        .addField("Coolguy3289#2290", "Developer // Website", false)
                        .addField("\uD83D\uDC7Bqu\uD83D\uDC7Brium#9352", "Developer // Website", false)
                        .build()
        ).queue();
    }
}

File: src/main/kotlin/me/diax/diax/commands/information/Credits.kt
package me.diax.diax.commands.information

import me.diax.comportment.jdacommand.Command
import me.diax.comportment.jdacommand.CommandDescription
import me.diax.diax.util.Emote.DRAGON_FACE
import me.diax.diax.util.Emote.MUSICAL_NOTE
import me.diax.diax.util.Emote.RAGE
import me.diax.diax.util.Emote.STAR
import me.diax.diax.util.style.Embed
import net.dv8tion.jda.core.entities.Message

@CommandDescription(
    name = "credits",
    triggers = ["credits"]
)
class Credits : Command {

    override fun execute(message: Message, s: String) {
        message.textChannel.sendMessage(
            Embed.themed()
                .setThumbnail(message.jda.selfUser.effectiveAvatarUrl)
                .addField("$STAR comportment#4475", "Owner // Head-Developer", false)
                .addField("$DRAGON_FACE AdrianTodt#0722", "Co-Owner // Backend-Developer", false)
                .addField("$RAGE Coolguy3289#2290", "Co-Owner // Website-Developer", false)
                .addField("$MUSICAL_NOTE NachtRaben#8307", "Ex-Developer // Music", false)
                .build()
        ).queue()
    }
}